docs/application/toc_all.md

```diff
@@ -223,53 +234,59 @@
 ###### [Text](/application/dotnet/guides/user-interface/nui/text.md)
 ###### [VideoView](/application/dotnet/guides/user-interface/nui/videoview.md)
 ###### [WebView](/application/dotnet/guides/user-interface/nui/webview.md)
-###### [AlertDialog](/application/dotnet/guides/user-interface/nui/nui-components/AlertDialog.md)
 ###### [Button](/application/dotnet/guides/user-interface/nui/nui-components/Button.md)
+###### [CameraView](/application/dotnet/guides/user-interface/nui/nui-components/CameraView.md)
 ###### [CheckBox](/application/dotnet/guides/user-interface/nui/nui-components/CheckBox.md)
-###### [DatePicker](/application/dotnet/guides/user-interface/nui/nui-components/DatePicker.md)
 ###### [Loading](/application/dotnet/guides/user-interface/nui/nui-components/Loading.md)
 ###### [LottieAnimationView](/application/dotnet/guides/user-interface/nui/nui-components/LottieAnimationView.md)
-###### [Menu](/application/dotnet/guides/user-interface/nui/nui-components/Menu.md)
+###### [Notification](/application/dotnet/guides/user-interface/nui/nui-components/Notification.md)
 ###### [Pagination](/application/dotnet/guides/user-interface/nui/nui-components/Pagination.md)
-###### [Picker](/application/dotnet/guides/user-interface/nui/nui-components/Picker.md)
 ###### [Popup](/application/dotnet/guides/user-interface/nui/nui-components/Popup.md)
 ###### [Progress](/application/dotnet/guides/user-interface/nui/nui-components/Progress.md)
 ###### [RadioButton](/application/dotnet/guides/user-interface/nui/nui-components/RadioButton.md)
 ###### [ScrollableBase](/application/dotnet/guides/user-interface/nui/nui-components/ScrollableBase.md)
 ###### [Slider](/application/dotnet/guides/user-interface/nui/nui-components/Slider.md)
 ###### [Switch](/application/dotnet/guides/user-interface/nui/nui-components/Switch.md)
 ###### [Tab](/application/dotnet/guides/user-interface/nui/nui-components/Tab.md)
-###### [TimePicker](/application/dotnet/guides/user-interface/nui/nui-components/TimePicker.md)
-##### [Layer](/application/dotnet/guides/user-interface/nui/layer.md)
+###### [Toast](/application/dotnet/guides/user-interface/nui/nui-components/Toast.md)
+
+##### [Layer](/application/dotnet/guides/user-interface/nui/layer.md)     
+
```

Comment:
Would you please rebase your PR to the latest? It seems PR 1522's update for toc_all.md was reverted here.